app/views/hello.scala.html

```diff
@@ -5,7 +5,7 @@
                 <h1>Hello World</h1>
             </article>
             <aside>
-                @commonSidebar()
+
```

Comment:
Was this function not defined before?

Reply:
I deleted it for the sake of the program going forward